polaris/ocean/tests/sphere_transport/resources/flow_types.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+from numpy import cos, pi, sin
+
+from polaris.ocean.tests.sphere_transport.resources.utils import (
+    calc_local_east_north,
+    lonlat2xyz,
+)
+
+
+def flow_nondivergent(t, lon, lat):
+    s_day = 3600. * 24.
+    twelve_days = s_day * 12.
+    lon_p = lon - 2. * pi * t / twelve_days
+    coslat = cos(lat)
+    cost = cos(pi * t / twelve_days)
+    u = (1 / twelve_days) * (10 * (sin(lon_p)**2) * sin(2 * lat) * cost +
+                             2. * pi * coslat)
+    v = (10 / twelve_days) * sin(2 * lon_p) * coslat * cost
+    return u, v
+
+
+def flow_divergent(t, lon, lat):
+    s_day = 3600. * 24.
+    twelve_days = 12. * s_day
+    lon_p = lon - 2. * pi * t / twelve_days
+    coslat = cos(lat)
+    cost = cos(pi * t / twelve_days)
+    u = (1 / twelve_days) * (-5 * (sin(lon_p / 2)**2) * sin(2 * lat) *
+                             (coslat**2) * cost + 2. * pi * coslat)
+    v = (2.5 / twelve_days) * sin(lon_p) * (coslat**3) * cost
+    return u, v
+
+
+def flow_rotation(lon, lat):
+    s_day = 3600. * 24.
+    Omega = [0.2, 0.7, 1.0]  # Constant rotation vector
+    Omega = (1 * pi / 12. * s_day) * (Omega / np.linalg.norm(Omega))
+    x, y, z = lonlat2xyz(lon, lat)
+    xyz = np.stack((x, y, z), axis=1)
+    vel = np.cross([0.2, 0.7, 1.0], np.transpose(xyz), axis=0)
```

Comment:
Based on this line:
https://github.com/E3SM-Project/E3SM/blob/master/components/mpas-ocean/src/shared/mpas_ocn_transport_tests.F#L396
I think this should be:
```suggestion
    vel = np.cross(Omega, np.transpose(xyz), axis=0)
```